data/json/mutations/mutations.json

```diff
@@ -2830,16 +2832,135 @@
     "type": "mutation",
     "id": "GIZZARD",
     "name": "Gizzard",
-    "points": -5,
-    "description": "You seem to get full faster now, and food goes through you more rapidly as well.",
+    "points": 2,
+    "description": "Your digestive tract has grown a gizzard, a small compartment attached to the stomach that helps digest food.  Slightly increases the energy gained through eating.",
     "valid": false,
     "purifiable": false,
     "prereqs": [ "BEAK", "BEAK_PECK", "BEAK_HUM" ],
-    "prereqs2": [ "LIGHTEATER" ],
     "threshreq": [ "THRESH_BIRD" ],
-    "cancels": [ "GOURMAND" ],
+    "leads_to": [ "BIRDCROP" ],
+    "category": [ "BIRD" ]
+  },
+  {
+    "type": "mutation",
+    "id": "FLEET3",
+    "name": "Subsonic-Sprinter",
+    "points": 5,
+    "description": "Your legs move at incredible speeds!  You move 40% faster on flat surfaces.",
+    "prereqs": [ "FLEET2" ],
+    "types": [ "RUNNING" ],
+    "category": [ "BIRD" ],
+    "threshreq": [ "THRESH_BIRD" ],
+    "movecost_flatground_modifier": 0.6
+  },
+  {
+    "type": "mutation",
+    "id": "BIRDCROP",
+    "name": "Bird Crop",
+    "points": 3,
+    "description": "Your throat has grown a bird crop, allowing you to store a little excess food if you overeat.",
+    "valid": false,
+    "purifiable": false,
+    "prereqs": [ "GIZZARD" ],
+    "threshreq": [ "THRESH_BIRD" ],
+    "category": [ "BIRD" ]
+  },
+  {
+    "type": "mutation",
+    "id": "WINGS_FOLDING",
+    "name": "Folding Wings",
+    "points": 3,
+    "visibility": 4,
+    "ugliness": 2,
+    "purifiable": false,
+    "description": "Your wings are flexible now, allowing you to fold them against your body.  They no longer encumber you when uncovered, and negate fall damage.",
+    "types": [ "WINGS" ],
+    "prereqs": [ "WINGS_BIRD" ],
+    "threshreq": [ "THRESH_BIRD" ],
+    "encumbrance_covered": [ [ "TORSO", 4 ] ],
+    "armor": [ { "parts": [ "TORSO" ], "bash": 2 } ],
```

Comment:
Those wings are on the torso? Like an angel? Bird wings are literally arms though